contributing.md

```diff
@@ -0,0 +1,54 @@
+# Contributing
+
+Welcome to the Pelican project! We are excited to have you contribute to our open-source project. This guide will help you get started with setting up your development environment, understanding our coding standards, and making your first or next contribution.
+
+## Getting started
+
+To start contributing to Pelican Panel, you need to have a basic understanding of the following:
+
+* [PHP](https://php.net) & [Laravel](https://laravel.com)
+* [Livewire](https://laravel-livewire.com) & [Filament](https://filamentphp.com)
+* [Git](https://git-scm.com) & [Github](https://github.com)
+
+## Dev Environment Setup
+
+1. Fork the Repository
+2. Clone your Fork
+3. Install Dependencies (PHP modules & composer, and run `composer install`)
```

Comment:
```suggestion
3. Install Dependencies (PHP modules & composer, and run `composer install --optimize-autoloader`)
```

Reply:
No, in development you should not optimize the autoloader. https://getcomposer.org/doc/articles/autoloader-optimization.md#autoloader-optimization